controllers/deleteQuestion.js

```diff
@@ -0,0 +1,20 @@
+const models = require("../models/index");
+const deleteQuestion = async (req, res) => {
+
+    try {
+        const questionId = parseInt(req.params.id);
+        const userId = req.user_id;
+
+        const count = await models.questions.destroy({
+            where: { id: questionId, user_id: userId }
+        });
+
+        return res.status(200).json({ success: 'true', message: 'successfully deleted a question' })
+
+    } catch (err) {
+        return res.status(401).json({ success: 'false', message: 'unauthorized to delete this question' })
+    }
```

Comment:
Returning `401` status here is not the best. It will mislead the user consuming your API, you should return `500`. Also the error message should be something generic like "An error occurred while processing your request" or you simply return the `err.message`